.github/workflows/veracode-analysis.yml

```diff
@@ -140,20 +142,79 @@ jobs:
           key: "${{ inputs.module_name }}-${{ github.sha }}-${{ github.run_id }}-veracode-binary"
 
-      - name: Run a sandbox scan
+      - name: Sandbox scan
         uses: veracode/veracode-uploadandscan-action@master
+        continue-on-error: true
         with:
           appname: "${{ inputs.module_name }}"
-          version: "${{ inputs.major_version }}.${{ inputs.minor_version }}_sha-${{ github.sha }}"
+          version: "${{ inputs.major_version }}.${{ inputs.minor_version }}_runId-${{ github.run_id }}"
           filepath: "${{ inputs.module_directory }}/${{ inputs.module_name }}-${{ github.sha }}-${{ github.run_id }}-veracode-binary.zip"
           vid: '${{ secrets.veracode_api_id }}'
           vkey: '${{ secrets.veracode_api_key }}'
           createprofile: true
           createsandbox: true
           sandboxname: "${{ github.ref_name }}"
-          criticality: "High"
+          criticality: "Low"
           scantimeout: 60
           includenewmodules: true
           scanallnonfataltoplevelmodules: true
           deleteincompletescan: 1
           scanpollinginterval: 60 # time between two checks in seconds / [30 to 120]
+
+  sca-scan:
+    needs: [build]
+    name: Run a SCA scan
+    # only stable and unstable maintenances branches will produce a report
+    if: needs.build.outputs.development_stage != 'Development'
+    runs-on: ubuntu-latest
+    continue-on-error: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Preparing dependencies
+        run: |
+          cd ${{ inputs.module_directory }}
+          REINSTALL_PACKAGE_JSON='true'
+
+          if [[ -f package-lock.json ]]; then
+            echo "[DEBUG] - Found a 'package-lock.json'"
+            RESULT=0
+            npm ci --legacy-peer-dep --omit=dev 1>/dev/null || RESULT=$( echo $? )
+            if [[ $RESULT -ne 0 ]]; then
+              echo "[DEBUG] - Failure of '$ npm ci'. Deleting the 'package-lock.json' to generate a new one"
+              rm -f package-lock.json
+            else
+              echo "[DEBUG] - 'package-lock.json' seems ready for analysis"
+              REINSTALL_PACKAGE_JSON='false'
+            fi
+          fi
+
+          if [[ -f package.json && REINSTALL_PACKAGE_JSON != 'false' ]]; then
+            echo "[DEBUG] - Found a 'package.json'"
+            RESULT=0
+            echo "[DEBUG] - Generate a 'package-lock.json'"
+            npm i --package-lock-only --legacy-peer-dep --omit=dev 1>/dev/null || RESULT=$( echo $? )
+            if [[ $RESULT -ne 0 ]]; then
+              echo "[DEBUG] - Failure of '$ npm install'. Deleting the 'package.json' to avoid analysis failure"
+              rm -f package.json
+            else
+              echo "[DEBUG] - 'package-lock.json' ready for analysis"
+            fi
+          fi
+
+      - name: SCA scan
+        env:
+          SRCCLR_API_TOKEN: ${{ secrets.veracode_srcclr_token }}
```

Comment:
that may be a detail but: why is this value called "SRCCLR_API_TOKEN" in this worfklow's environment whereas it's called "veracode_srcclr_token" is other environments?